packages/annotorious-core/src/model/Annotator.ts

```diff
@@ -145,16 +145,16 @@ export const createBaseAnnotator = <I extends Annotation, E extends unknown>(
     }
   }
 
-  const setAnnotations = (annotations: E[]) => {
+  const setAnnotations = (annotations: E[], replace = true) => {
```

Comment:
`replace` is also `true` by default in the `bulkAddAnnotation`:
https://github.com/oleksandr-danylchenko/annotorious/blob/8de692434213ad910df3af53fef28b620c60ffc9/packages/annotorious-core/src/state/Store.ts#L136